src/sentry/api/endpoints/auth_index.py

```diff
@@ -111,9 +117,12 @@ def _require_password_or_u2f_check():
 
         if _require_password_or_u2f_check():
             authenticated = self._verify_user_via_inputs(validator, request)
-
+        breakpoint()
```

Comment:
```suggestion
```

We probably dont want this in prod :D

Reply:
yeah i just got rid of it :c I have no idea why but usually that is caught